frontend/src/main.ts

```diff
@@ -19,9 +19,20 @@ import vuetifyOptions from "@/vuetifyOptions";
 import "vuetify/dist/vuetify.min.css";
 
 import { rtdbPlugin as VueFire } from "vuefire";
+import VuePageTitle from "vue-page-title";
 
 import "@babel/polyfill";
 
+Vue.use(VuePageTitle, {
+  suffix: "| Audio Seva",
+  setTitleMethod: (title: string) => {
+    if (title.includes("Home |")) {
+      document.title = "Audio Seva";
+    } else {
+      document.title = title;
+    }
```

Comment:
Here I'd assume that the title is empty/null rather than `Home |`. You can compare the title to the suffix and if they match then trim the pipe from the suffix. This way you'd avoid duplication of `Home` and of `Audio Seva`.

Reply:
If no title is used in Home the function is never called.

Reply:
That means that the page title will remain the same as on the previous route?

By the way, what do you mean by “no title is used”? For `undefined` it may be true, but what about `null` or empty?

Reply:
Yes the title stays the same as the previous page. This behavior is there if title is undefined, null or empty.